README.md

```diff
@@ -39,7 +39,21 @@ This is all possible inside `todo.txt`.
 
 ## `todo.txt` format rules
 
-<img src="./description.svg" width="100%" height="500">
+```
+┌─────────────────── Optional - Marks completion 
+│  ┌──────────────── Optional - Marks priority 
+│  │      ┌─────────────── Optional - Completion date 
+│  │      │          ┌──── Optional - Creation date
+│  │      │          │     (must be specified if completion date is)
+│  │      │          │             ┌─── Description. Tags (optional)
+│  │      │          │             │    can be placed anywhere in here 
```

Comment:
Great job, this is very comprehensive and easy to read :)

The description and tags part is the only one that wasn't immediately obvious to me. How about something like this?

```suggestion
│  │      │          │             ┌─── Mandatory - Description
│  │      │          │             │    Tags (optional) can be placed anywhere within it.
```

I think it makes it slightly clearer.

Reply:
Excellent!